src/coreclr/gc/unix/gcenv.unix.cpp

```diff
@@ -876,21 +876,29 @@ bool ReadMemoryValueFromFile(const char* filename, uint64_t* val)
     return result;
 }
 
+#define CHECK_CACHE_SIZE(CACHE_LEVEL) if (size > cacheSize) { cacheSize = size; cacheLevel = CACHE_LEVEL; }
```

Comment:
A nit - would you mind renaming this to `UPDATE_CACHE_SIZE` or `UPDATE_CACHE_SIZE_AND_LEVEL`? It would make it clearer what it does.

Reply:
Will do.